cli.js

```diff
@@ -1210,6 +1211,180 @@ program
+    .option('-c, --certificate <certificate>','Path to the certificate to use for two factor authentication.')
+    .option('-p, --passphrase <passphrase>','Passphrase to be used to read the given certificate.')
+    .option('-v, --verbose', 'Verbose mode')
+    .description('Generate a manifest for the given local directories. ' +
+        'Compare this manifest with the one within the active code version of the instance. ' +
+        'Deploy only the files which have been updated locally comparing to the remote, ' +
+        'within a newly created code version.' +
+        'Activate this newly generated code version if required in the options')
+    .action((codeversion, localdirectorypaths, options) => {
+        var instance = require('./lib/instance').getInstance(options.instance);
+        require('./lib/code').cli.diffdeploy(instance, localdirectorypaths, codeversion, {
+            sourceCodeVersion: options.sourcecodeversion,
+            manifestFileName: options.manifestfilename,
+            pfx: options.certificate,
+            passphrase: options.passphrase,
+            overrideLocalFile: options.override,
+            removeFilesAfter: options.removeafter,
+            ignorePatterns: options.ignore,
+            forceDeployPatterns: options.forcedeploy,
+            verbose: options.verbose
+        }, options.activate || false).catch(err => {
+            console.log(colors.red(err))
+            process.exit(-1);
+        });
+    }).on('--help', () => {
+        console.log();
+        console.log('  Details:');
+        console.log();
+        console.log('  This command performs a differential deployment by following these steps:');
+        console.log('  1. Get the active code version (if it does not exist or ' +
+            'there is an issue while connecting, abort)');
+        console.log('  2. Download the manifest file from the code version (if it does not exist, abort)');
+        console.log('  3. Generate a local manifest, which represents the state of the local files ' +
+            '(if there is any issue finding files, abort)');
+        console.log('  4. Compare both manifests, and keep track of the changed files. ' +
+            'If no files are changed, end the process');
+        console.log('  5. Copy the remote code version in a new folder, named with the codeversion parameter.');
+        console.log('  6. Upload files through a partial ZIP archive which contains the added and changed files, ' +
+            'based on the comparison done at step#4. The new manifest is also part of this archive. ' +
+            'For removed files, perform a DELETE request, one file at a time.');
+        console.log('  7. Activate the newly generated code version on the instance, ' +
+            'if the activate option is passed.');
+        console.log('  8. Remote the manifest and archive files locally after the process if the removeafter option is passed.');
```

Comment:
just a small type `removeafter`

Reply:
This is the name of the CLI option, which is why I didn't use a space here

Reply:
Thanks 👍